<launch>

    <node name="amigo_head_ref" pkg="amigo_head_ref" type="head_ref" output="screen">
        <param name = "min_pan" value="-2.681" />
        <param name = "max_pan" value="2.681" />
        <param name = "min_tilt" value="-0.3787" />
        <param name = "max_tilt" value="1.0236" />
        
        <param name = "max_vel_pan" value="0.2" />
        <param name = "max_vel_tilt" value="0.07" />
        <param name = "max_acc_pan" value="0.8" />
        <param name = "max_acc_tilt" value="0.8" />
        <param name = "frequency" value="50" />
        <param name = "goal_tolerance_pan" value="0.02" />
        <param name = "goal_tolerance_tilt" value="0.06" />
    </node>

 </launch>
  
